src/core/Button/style.ts

```diff
@@ -24,12 +24,16 @@ const ButtonBase = styled(Button, {
       padding: ${spacings?.xs}px ${spacings?.l}px;
       min-width: 120px;
       height: 34px;
-      &:hover {
+      &:hover, &:focus {
         color: white;
         background-color: ${colors?.primary[500]};
         border: none;
         box-shadow: none;
       }
+      &:focus {
+        outline: 5px auto Highlight;
```

Comment:
did not know about `Highlight`, very cool!

Reply:
ooh same!! Really cool 💡 👏

Reply:
me neither, someone from the internet use it 😅